pom.xml

```diff
@@ -128,24 +128,21 @@
     <aws.amazonaws.version>1.11.215</aws.amazonaws.version>
     <build.path>build</build.path>
     <copycat.version>1.2.14</copycat.version>
-    <cxf.version>2.7.0</cxf.version>
     <glusterfs-hadoop.version>2.3.13</glusterfs-hadoop.version>
-    <gmetric4j.version>1.0.7</gmetric4j.version>
     <grpc.version>1.27.0</grpc.version>
     <netty.version>4.1.42.Final</netty.version>
     <hadoop.version>2.7.3</hadoop.version>
-    <hadoop-openstack.version>2.6.0</hadoop-openstack.version>
     <jacoco.version>0.8.5</jacoco.version>
     <java.version>1.8</java.version>
-    <jersey.version>2.22</jersey.version>
-    <jetty.version>9.2.16.v20160414</jetty.version>
+    <jersey.version>2.29.1</jersey.version>
+    <jetty.version>9.2.28.v20190418</jetty.version>
     <junit.version>4.12</junit.version>
     <log4j.version>1.2.17</log4j.version>
     <maven.version>3.3.9</maven.version>
     <metrics.version>3.1.2</metrics.version>
     <oshi.version>4.2.0</oshi.version>
     <powermock.version>1.6.2</powermock.version>
-    <prometheus.version>0.3.0</prometheus.version>
+    <prometheus.version>0.8.0</prometheus.version>
```

Comment:
Are there any changes in the format in which metrics might be reported from 0.3 to 0.8? According to semver, Upgrades from `0.x` to `0.(x+1)` may not have a stable API

https://semver.org/#spec-item-4

Reply:
Mainly new features and bug fixes, it looks like there are no compatibility issues

https://github.com/prometheus/client_java/releases